### PR TITLE
Add Afterpay badge view and assets

### PR DESCRIPTION
**Pull request by Rypac**
## Summary of Changes

- Add Afterpay badge view badge and include in example app.

## Items of Note

Assets are currently placeholders.

Screenshot_1596412183

### CODE REVIEW

File: afterpay/src/main/kotlin/com/afterpay/android/view/AfterpayBadge.kt [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        layoutParams = LayoutParams(LayoutParams.WRAP_CONTENT, 48.dp).apply {
+            gravity = Gravity.CENTER
+        }
+    }
+
+    init {
+        contentDescription = resources.getString(R.string.badge_content_description)
+        importantForAccessibility = View.IMPORTANT_FOR_ACCESSIBILITY_YES
+        layoutParams = LayoutParams(LayoutParams.WRAP_CONTENT, LayoutParams.WRAP_CONTENT)
+        isClickable = true
```
Comment (adamjcampbell) [inline afterpay/src/main/kotlin/com/afterpay/android/view/AfterpayBadge.kt:37]: I'm not sure the badge is intended to be a pay button. More of a thing to mark a product as eligible for Afterpay

File: afterpay/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -6,4 +6,6 @@
     <string name="load_error_retry">Retry</string>
     <string name="load_error_cancel">Cancel</string>
 
+    <string name="badge_content_description">Afterpay it in 4. Always interest-free.</string>
```
Comment (adamjcampbell) [inline afterpay/src/main/res/values/strings.xml:9]: Is this for accessibility? I had trouble with the pronunciation of Afterpay and ended up using "after pay". On iOS you can input the IPA pronunciation for something. However this locks it into the American voice it seems and doesn't sound as natural.

I'm not 100% on what we want to use as the content string. Was this specified somewhere or should be raise it with the team?

Comment (Rypac) [reply afterpay/src/main/res/values/strings.xml:9]: Yeah, this is for accessibility. It reads Afterpay very naturally in the Australian voice but that's a good point, I'll test with other voices.

I'm not 100% sure either and this is basically the placeholder. I went with the phrase that is used almost exclusively when placed next to a badge in the branding guide.

File: afterpay/src/main/kotlin/com/afterpay/android/view/AfterpayBadge.kt (hunk 1 above)
Comment (Rypac) [reply afterpay/src/main/kotlin/com/afterpay/android/view/AfterpayBadge.kt:37]: Good catch! I meant to take that out. This is still hanging around from when it _was_ a button.

File: afterpay/src/main/res/values/strings.xml (hunk 2 above)
Comment (Rypac) [reply afterpay/src/main/res/values/strings.xml]: Changed to "After pay" to help the screen reader.

**Comment by Rypac** [comment]
@adamjcampbell I fixed up the issues you raised so it should be ready for another review now.

**Pull request merged**